### PR TITLE
feat: removed streaming

### PR DESCRIPTION
**Pull request by tstaetter**
feat: removed obsolete code, added node service

As preparation for introducing MQTT

### CODE REVIEW

**Comment by tstaetter** [comment]
🩸❤️

**Comment by loony-bean** [comment]
could be easier to migrate if we first add mqtt, and remove streaming after switch

**Comment by tstaetter** [comment]
i want the API to be able to work w/o MQTT

**Review by loony-bean — APPROVED** [review]

File: blockjoy/api/v1/command.proto [hunk 1]
```diff
@@ -2,10 +2,40 @@ syntax = "proto3";
...
+message PendingCommandsRequest {
+  string host_id = 1;
+  /* maybe use it for getting only specific type of pending commands? */
+  optional string filter_type = 2;
+}
+
+/* Commands service used in favour of deprecated command flow */
+service Commands {
+  /* Retrieve a single command */
+  rpc Get(CommandInfo) returns (Command);
```
Comment (mjr-blockjoy) [inline blockjoy/api/v1/command.proto:36]: I don't think we need it, as long as we want to process commands in order.

**Review by ThouCheese — APPROVED** [review]

**Pull request merged**